pa-rconnector/src/main/PARConnector/R/PA.r

```diff
@@ -454,7 +453,7 @@ PAS <- function(funcOrFuncName, ..., varies=NULL, input.files=list(), output.fil
   dots <- list(...)
    
   # we generate a task with a forced cardinality of 1
-  task <- PA(funcOrFuncName, ..., varies=list(),input.files=input.files, output.files=output.files, in.dir = in.dir, out.dir = out.dir, hostname.selection = hostname.selection, ip.selection = ip.selection, property.selection.name = property.selection.name, property.selection.value = property.selection.value, nodes.number = nodes.number, topology = topology, generic.information.list = generic.information.list, run.as.me = run.as.me, walltime = walltime, isolate.io.files = isolate.io.files, client = client, .debug = .debug)
+  task <- PA(funcOrFuncName, ..., varies=list(),input.files=input.files, output.files=output.files, in.dir = in.dir, out.dir = out.dir, hostname.selection = hostname.selection, ip.selection = ip.selection, property.selection.name = property.selection.name, property.selection.value = property.selection.value, nodes.number = nodes.number, topology = topology, generic.information.list = generic.information.list, run.as.me = run.as.me, walltime = walltime, isolate.io.files = isolate.io.files, client = client, .debug = .debug, .notransfer = FALSE)
```

Comment:
Why setting .notransfer to FALSE by default?

Reply:
yes, this is a complicate matter.

In the PAM code (merge function), a for loop calls, for each parent, the PA function. These PA calls will not actually create real ProActive workflow tasks, but are only here to analyse the output files from parent R functions.
I introduced the .notransfer option to disable in these calls file transfer and avoid having R data files transferred to the server for tasks which are never going to be executed.